Assets/BossRoom/Scripts/Client/UI/UIRadioButton.cs

```diff
@@ -0,0 +1,51 @@
+using System;
```

Comment:
As discussed, let's try using toggle groups instead of having our own implementation

Reply:
Done.